source/change-streams/change-streams.rst

```diff
@@ -433,7 +433,7 @@ Drivers MUST use the ``ns`` returned in the ``aggregate`` command to set the ``c
 ChangeStream
 ------------
 
-A ``ChangeStream`` is an abstraction of a `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_ cursor, with support for resumability.  Implementors MAY choose to implement a ``ChangeStream`` as an extension of an existing tailable cursor implementation.  If the ``ChangeStream`` is implemented as a type which owns a tailable cursor, then the implementor MUST provide a manner of closing the change stream, as well as satisfy the requirements of extending ``Iterable<Document>``. In languages with more idiomatic patterns of closing, such as those with RAII or destructors, drivers may use those patterns instead of a close method. 
+A ``ChangeStream`` is an abstraction of a `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_ cursor, with support for resumability.  Implementors MAY choose to implement a ``ChangeStream`` as an extension of an existing tailable cursor implementation.  If the ``ChangeStream`` is implemented as a type which owns a tailable cursor, then the implementor MUST provide a manner of closing the change stream, as well as satisfy the requirements of extending ``Iterable<Document>``. In languages with more idiomatic patterns of closing, such as those with RAII or destructors, drivers MAY use those patterns instead of a close method. 
```

Comment:
Is the only change here that MAY is now capitalized (to use RFC terms) in "drivers may use"?

Reply:
On the specific commit you commented on, yes. But the other commit in the PR adds the language about the `close` method requirement being fulfilled by RAII/destructors.

Reply:
This reads a bit awkwardly to me, how about borrowing from the sessions spec, something like:
```
If your language has an idiomatic way of disposing resources you MAY choose to implement that in addition to, or instead of, an explicit close method, whichever is more idiomatic for your language.
```

Reply:
I agree that this sounds more natural/explicit. Used this phrase (changed `disposing resources` to `disposing of resources`, though).